### PR TITLE
Task page

### PR DESCRIPTION
**Pull request by junaidshaikh-js**
Add static UI for task page 

- The page is responsive
- Contains a header which wishes the user and tells about the number of tasks for a particular day
- The task container contains a button to add a new task and list existing tasks.

🔗  Preview: https://deploy-preview-8--clockwork-app.netlify.app/task

### CODE REVIEW

File: src/components/task/TaskCard.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { FaEdit, FaTrashAlt } from "react-icons/fa";
+
+export function TaskCard({ title }) {
+  return (
+    <section className="task-card flex justify-between align-center">
+      <p className="task-title">{title}</p>
+      <div className="mr-1">
+        <FaEdit style={{ marginRight: "1rem", cursor: "pointer" }} />
+        <FaTrashAlt style={{ cursor: "pointer" }} />
```
Comment (adith-a-danthi) [inline src/components/task/TaskCard.jsx]: Avoid inline styles. You can create CSS classes for the same

**Review by adith-a-danthi — COMMENTED** [review]
Clean, well-structured code. Good to merge

File: src/components/task/TaskCard.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { FaEdit, FaTrashAlt } from "react-icons/fa";
+
+export function TaskCard({ title }) {
+  return (
+    <section className="task-card flex justify-between align-center">
+      <p className="task-title">{title}</p>
+      <div className="mr-1">
+        <FaEdit style={{ marginRight: "1rem", cursor: "pointer" }} />
```
Comment (shivampandey0) [inline src/components/task/TaskCard.jsx]: utility classes can be used instead of inline CSS?

**Review by shivampandey0 — APPROVED** [review]
Semantic tags are used with individual react components to form the page.
good to merge

File: src/components/task/TaskCard.jsx (hunk 2 above)
Comment (junaidshaikh-js) [reply src/components/task/TaskCard.jsx]: Changes made.

File: src/components/task/TaskCard.jsx (hunk 1 above)
Comment (junaidshaikh-js) [reply src/components/task/TaskCard.jsx]: Changes made.

**Pull request merged**